uml/design.uml

```diff
@@ -69,11 +87,20 @@ class Hand {
     discard()
 }
 
+class Card {
+    type
+    mana_cost
+    overload
+
```

Comment:
We should probably add play[_from_hand]() method to this abstract class since there is no apparent difference between playing spell, minion or weapon - some action with requirements and consequences.